.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: [ main ]
+
+jobs:
+  lint-test-docs:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: ['3.9', '3.13']
+        os: [ubuntu-latest, windows-latest]
```

Comment:
I've temporarily reduced the matrix size because GH was refusing to run the jobs.